internal/cli/start.go

```diff
@@ -28,9 +29,17 @@ var startCmd = &cobra.Command{
 		if provider == nil {
 			log.Fatal().Msgf("provider %s not found", options.ProviderName)
 		}
-		if err := provider.Start(); err != nil {
+
+		template, err := templates.GetCurrentTemplate()
```

Comment:
It would be nice to rename GetCurrentTemplate to something else, can be confusing.